packages/compiler-sfc/src/rewriteDefault.ts

```diff
@@ -42,7 +42,12 @@ export function rewriteDefault(
   }).program.body
   ast.forEach(node => {
     if (node.type === 'ExportDefaultDeclaration') {
-      s.overwrite(node.start!, node.declaration.start!, `const ${as} = `)
+      if (node.declaration.type === 'ClassDeclaration') {
+        s.overwrite(node.start!, node.declaration.id.start!, `class `)
```

Comment:
Might want to remove the `!` here?

Reply:
`node.start` is a `Nullable<number>`, so that we need `!`